instrumentation/opentelemetry-instrumentation-botocore/tests/conftest.py

```diff
@@ -118,4 +118,85 @@ def scrub_response_headers(response):
     This scrubs sensitive response headers. Note they are case-sensitive!
     """
     response["headers"]["Set-Cookie"] = "test_set_cookie"
+    # SDK validates response length which we modify by pretty-printing JSON.
+    # Content length does not affect our instrumentation so just drop it.
+    response["headers"].pop("Content-Length", None)
     return response
+
+
+class LiteralBlockScalar(str):
+    """Formats the string as a literal block scalar, preserving whitespace and
+    without interpreting escape characters"""
+
+
+def literal_block_scalar_presenter(dumper, data):
+    """Represents a scalar string as a literal block, via '|' syntax"""
+    return dumper.represent_scalar("tag:yaml.org,2002:str", data, style="|")
+
+
+yaml.add_representer(LiteralBlockScalar, literal_block_scalar_presenter)
+
+
+def process_string_value(string_value):
+    """Pretty-prints JSON or returns long strings as a LiteralBlockScalar"""
+    if isinstance(string_value, bytes):
```

Comment:
I added this to continue rendering binary as `!!binary` base64 rather than `b'\x00\x01...'`

Reply:
I think this will result in less huge data, and will be familiar to people who will see base64 more often than `b'\x00\x01...'`